### PR TITLE
enable right-handed normal maps

### PR DESCRIPTION
**Pull request by roudy16**
Addresses #313

### CODE REVIEW

**Comment by roudy16** [comment]
I think this is the least invasive way to use the normal map coordinate system you want, another approach would be to update all the shaders to flip the Y value.

File: code/intermediate/tutorial11-normals/src/model.rs [hunk 1]
```diff
@@ -220,6 +220,9 @@ impl Model {
                 let r = 1.0 / (delta_uv1.x * delta_uv2.y - delta_uv1.y * delta_uv2.x);
                 let tangent = (delta_pos1 * delta_uv2.y - delta_pos2 * delta_uv1.y) * r;
                 let bitangent = (delta_pos2 * delta_uv1.x - delta_pos1 * delta_uv2.x) * r;
+                // We flip the bitangent to enable right-handed normal
```
Comment (roudy16) [inline code/intermediate/tutorial11-normals/src/model.rs:223]: @sotrh Are you happy with this comment and the use of the `flipped_bitangent` var?

File: docs/intermediate/tutorial11-normals/README.md [hunk 2]
```diff
@@ -279,6 +279,9 @@ impl Model {
                 let r = 1.0 / (delta_uv1.x * delta_uv2.y - delta_uv1.y * delta_uv2.x);
                 let tangent = (delta_pos1 * delta_uv2.y - delta_pos2 * delta_uv1.y) * r;
                 let bitangent = (delta_pos2 * delta_uv1.x - delta_pos1 * delta_uv2.x) * r;
+                // We flip the bitangent to enable right-handed normal
+                // maps with wgpu texture coordinate system
+                let flipped_bitangent = bitangent * -1.0;
```
Comment (sotrh) [inline docs/intermediate/tutorial11-normals/README.md:284]: I'd probably just add the -1.0 to the `let bitangent = ...` line instead, and then add a comment above stating that we're using a right handed coordinate system.

Alternatively you could also flip the subtraction. Something like:
```rust
let bitangent = (delta_pos1 * delta_uv2.x - delta_pos2 * delta_uv1.y) * r;
```

**Review by sotrh — CHANGES_REQUESTED** [review]

File: code/intermediate/tutorial11-normals/src/model.rs (hunk 1 above)
Comment (roudy16) [reply code/intermediate/tutorial11-normals/src/model.rs:222]: updated to not introduce new var.

File: docs/intermediate/tutorial11-normals/README.md (hunk 2 above)
Comment (roudy16) [reply docs/intermediate/tutorial11-normals/README.md]: @sotrh updated.

**Review by sotrh — APPROVED** [review]

**Pull request merged**